aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -37,3 +41,14 @@ impl PackageHooks for AptosPackageHooks {
         bail!("not used")
     }
 }
+
+pub(crate) fn build_package(
+    package_path: PathBuf,
+    options: BuildOptions,
+) -> anyhow::Result<BuiltPackage> {
+    let mut options = options;
+    if read_env_var(ENABLE_V2) == "1" {
```

Comment:
Can we please use `env::read_bool_env_var` instead in case people use "=true" instead of "=1"?

Reply:
done